examples/transfer_from_opentx.rs

```diff
@@ -0,0 +1,722 @@
+/*
+How to use the example transfer_from_opentx, see the file transfer_from_opentx.md
+*/
+use ckb_crypto::secp::Pubkey;
+use ckb_hash::blake2b_256;
+use ckb_jsonrpc_types as json_types;
+use ckb_sdk::{
+    constants::SIGHASH_TYPE_HASH,
+    rpc::CkbRpcClient,
+    traits::{
+        DefaultCellCollector, DefaultCellDepResolver, DefaultHeaderDepResolver,
+        DefaultTransactionDependencyProvider, SecpCkbRawKeySigner,
+    },
+    tx_builder::{
+        balance_tx_capacity, fill_placeholder_witnesses, omni_lock::OmniLockTransferBuilder,
+        unlock_tx, CapacityBalancer, TxBuilder,
+    },
+    types::NetworkType,
+    unlock::{
+        opentx::{assembler::assemble_new_tx, OpentxWitness},
+        IdentityFlag, MultisigConfig, OmniLockConfig, OmniLockScriptSigner, SecpSighashUnlocker,
+    },
+    unlock::{OmniLockUnlocker, OmniUnlockMode, ScriptUnlocker},
+    util::{blake160, keccak160},
+    Address, HumanCapacity, ScriptGroup, ScriptId, SECP256K1,
+};
+use ckb_types::{
+    bytes::Bytes,
+    core::{BlockView, Capacity, ScriptHashType, TransactionView},
+    packed::{Byte32, CellDep, CellOutput, OutPoint, Script, Transaction, WitnessArgs},
+    prelude::*,
+    H160, H256,
+};
+use clap::{Args, Parser, Subcommand};
+use serde::{Deserialize, Serialize};
+use std::{collections::HashMap, error::Error as StdErr, fs, path::PathBuf};
+
+const OPENTX_TX_HASH: &str = "d7697f6b3684d1451c42cc538b3789f13b01430007f65afe74834b6a28714a18";
```

Comment:
Describe where this tx hash comes from, and the impact to which CKB network this example should connect to.